source/includes/sample-data-note-plural.rst

```diff
@@ -0,0 +1,9 @@
+.. note::
```

Comment:
I think these notes are repeating content currently outlined in the Quick Start section so maybe we could link to that instead. There might be too much instruction for a reader to reasonably follow when they are trying to focus on how to build their query.

I wonder if it might be easier to either:
1) mention that they can run the examples (but not as-is because they don't feature the setup code) if they've set up their Atlas and sample data as explained in the linked quick start.
2) just link to the document schema of the sample data which is conceptually similar to how we might present sample documents for content that follows.

Reply:
Thanks for the note. In implementing feedback from https://github.com/mongodb/docs-node/pull/167#discussion_r623382464 I ended up deleting the note. My edits linked to the quick start informed by your suggestion in this comment. I rewrote the notes to contain less hyperlinks and be more centered around the quick start. While they are no longer featured on the page, if you see any issues with them I'd appreciate your feedback as I may use these notes elsewhere (these notes were originally from the Go docs but I think I prefer the rewrites here). 

Fix: https://github.com/mongodb/docs-node/pull/167/commits/076e84a09380ed5d802958925595aa7405d86b77

Going forward, I will try to ask myself the question, "is this a reasonable amount of information for a reader to follow?" I don't think I have this at the forefront of my mind enough when writing.

Reply:
For clarification, in point 2, I wasn't suggesting that you create a sample "toy dataset". I was suggesting you show the document schema. A quick google search reveals this example of a document schema example from Realm docs:
https://docs.mongodb.com/realm/mongodb/document-schemas/#schema-enforcement-process

The current fix looks good to me.

Reply:
Thank you for clarifying. I did not think to incorporate a document schema into a docs page as done in this example from the realm docs: https://docs.mongodb.com/realm/mongodb/document-schemas/#schema-enforcement-process. 

Going forward, I will keep presenting the schema of the collection I am referencing in mind as a possibility when setting up background for examples.